css-four-column-layout/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>CSS four column layout</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS four column layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+  <div class="row">
+    <div class="column-full">
+      <h2>How It Works</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="column-full">
```

Comment:
Instead of one `column-full`, you should make four `column-fourth` elements, and create a class with appropriate styling for it